src/components/SendTokens/EthereumTokens.js

```diff
@@ -449,15 +456,15 @@ const SendEthereumTokens = ({
         },
       }}
       customValueSelectorProps={{
-        getFormValue: handleAmountChange,
-        getError: manageFormErrorState,
+        value: amount,
+        onValueChange: setAmount,
+        assetData,
+        onAssetDataChange: setAssetData,
+        showCollectibles: true,
         txFeeInfo,
-        preselectedAsset: token,
-        preselectedCollectible,
-        showAllAssetTypes: true,
-        gettingFee,
         hideMaxSend: gettingFee || !selectedContact, // we cannot calculate max if no receiver is set
```

Comment:
I guess, this (`// we cannot calculate max if no receiver is set`) is no longer valid, isn't it?